assets/_common/styles/custom-styles.scss

```diff
@@ -42,3 +42,10 @@ h1, h2, h3, h4, h5, h6 {
     margin-bottom: 0;
   }
 }
+
+// Widen header title
+@media (min-width: 64em) {
+  .usa-header--extended .usa-logo {
+    max-width: 70%;
```

Comment:
Where possible I like to avoid unnamed values like this `70%`.  I suggest USWDS's max width include or utility: here like `@include u-maxw('tablet')` or `max-width: u-maxw('tablet')`

https://designsystem.digital.gov/utilities/height-and-width/ for more info

Reply:
Turns out the original CSS I'm trying to overwrite here is controlled by the `$theme-header-logo-text-width` variable. But using `$theme-header-logo-text-width: u-maxw('tablet'),` in `_uswds-theme-settings.scss` doesn't work. What should the syntax be instead?

(Although hilariously, using it anyway fixes the original problem since I've overwritten the original setting even if it doesn't register! 😅)

Reply:
Ha! For settings ironically it looks like they do want a percentage, so you could add in your 70% there.  My thinking is since the settings are very similar to defining a named variable, you could put in the raw 70% number there.